packages/admin/admin/src/index.ts

```diff
@@ -103,7 +103,8 @@ export { MasterLayoutClassKey } from "./mui/MasterLayout.styles";
 export { MasterLayoutContext } from "./mui/MasterLayoutContext";
 export { MenuCollapsibleItem, MenuCollapsibleItemClassKey, MenuCollapsibleItemProps, MenuLevel } from "./mui/menu/CollapsibleItem";
 export { IMenuContext, IWithMenu, MenuContext, withMenu } from "./mui/menu/Context";
-export { MenuItem, MenuItemClassKey, MenuItemProps } from "./mui/menu/Item";
+export { MenuItem, MenuItemProps } from "./mui/menu/Item";
+export { MenuItemClassKey } from "./mui/menu/Item.styles";
```

Comment:
Note: the theming of components is currently refactored in #1376, removing .styles files and the `createStyles` usage. The theme refactor will presumably be finished before the menu rework, so you'll have to change the styles in this PR.

Reply:
I think this should be done in a separate PR in the feature branch, once #1376 has been merged.

Reply:
Agree 👍🏻